.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,36 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Issue related to manifests
+    url: https://github.com/kubeflow/manifests/issues/new
+    about: Create issue in kubeflow/manfiests repo
+  - name: Issue related to deployment
```

Comment:
nit: `Issues related to kfctl`?

I think that makes users easier to identify the place to go.